docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/Ranking/PermutationFeatureImportance.cs

```diff
@@ -78,14 +88,25 @@ private class Data
         /// Generate an enumerable of Data objects, creating the label as a simple
         /// linear combination of the features.
         /// </summary>
+        /// 
         /// <param name="nExamples">The number of examples.</param>
-        /// <param name="bias">The bias, or offset, in the calculation of the label.</param>
-        /// <param name="weight1">The weight to multiply the first feature with to compute the label.</param>
-        /// <param name="weight2">The weight to multiply the second feature with to compute the label.</param>
-        /// <param name="seed">The seed for generating feature values and label noise.</param>
+        /// 
+        /// <param name="bias">The bias, or offset, in the calculation of the label.
+        /// </param>
+        /// 
```

Comment:
Empty lines floating around.

Reply:
I think for lines 93-105, the empty lines were there originally. Would you like me to get rid of them?